.github/copilot-instructions.md

```
@@ -83,30 +95,34 @@ Refer to specific instruction files in `.github/instructions/` for detailed guid
 - Node.js 22+ is required (check `.node-version` file)
 - Install Node.js 22: `curl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash - && sudo apt-get install -y nodejs`
 - Or use nvm: `nvm install 22 && nvm use 22`
 
 ### Bootstrap, Build, and Test the Repository
-- `npm install --ignore-scripts` -- installs dependencies without Cypress binary (takes ~16 seconds)
+
+- `npm install --ignore-scripts` -- installs dependencies without Playwright browsers (takes ~16 seconds)
 - `npm run postinstall` -- installs platform-specific dependencies and generates headers (takes ~3 seconds)
 - `npm run setup` -- generates plugin map and setup (takes ~1 second)
 - `npm run build` -- production build. NEVER CANCEL: Takes 2+ minutes. Set timeout to 180+ seconds.
 - `npm run dev` -- starts development server on http://localhost:4000 (takes ~5 seconds)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)